projects/js-packages/components/components/record-meter-bar/test/component.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
```

Comment:
We're in a movement out of `enzyme` for `react-testing-library`. Since this is being introduced now, can you move for it?

Context: p9dueE-3ZX-p2

Reply:
Thank you. I didn't know that. I have made the changes.